newshomepages/sources/sites.csv

```diff
@@ -75,3 +75,4 @@ BonginoReport,https://bonginoreport.com/,Bongino Report,,,,America/New_York,us-r
 BreitbartNews,https://www.breitbart.com/,Breitbart,,,,America/New_York,us-right-wing
 HumanEvents,https://humanevents.com/,Human Events,,,,America/New_York,us-right-wing
 OANN,https://www.oann.com/,One America News Network,,,,America/New_York,us-right-wing
+Carolina Public Press, https://carolinapublicpress.org/, Carolina Public Press,,,,America/New_York, north-carolina
```

Comment:
```suggestion
CPublicPress, https://carolinapublicpress.org/, Carolina Public Press,,,,America/New_York, north-carolina
```
This first field is intended to be the Twitter handle.

Reply:
ah, sorry! I'll do the rest following this convention.